cmd/rest/client.go

```diff
@@ -66,7 +66,9 @@ func (c *Client) Call(method string, values url.Values, body io.Reader, length i
 		if err != nil {
 			return nil, err
 		}
-		return nil, errors.New(string(b))
+		if len(b) > 0 {
+			return nil, errors.New(string(b))
```

Comment:
wait are we saying we won't return an error if `len(b) > 0` is not true?

Reply:
> wait are we saying we won't return an error if `len(b) > 0` is not true?

Good catch. I've fixed it, it was a last minute mistake.